.claude/skills/discord-oauth2.md

```
@@ -0,0 +1,151 @@
+---
+description: Architecture and implementation details of Discord OAuth2 + JWT session system
+---
+
+# Discord OAuth2 + JWT Session System
+
+## Full Authentication Flow
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)